ntoskrnl/config/i386/cmhardwr.c

```diff
@@ -251,6 +252,11 @@ CmpInitializeMachineDependentConfiguration(IN PLOADER_PARAMETER_BLOCK LoaderBloc
     extern UNICODE_STRING KeRosProcessorName, KeRosBiosDate, KeRosBiosVersion;
     extern UNICODE_STRING KeRosVideoBiosDate, KeRosVideoBiosVersion;
 
+    /* Set the alternative system architecture information */
```

Comment:
Technically this is done in CmpInitializeHardwareConfiguration, in case the configuration tree contains the "NEC PC-98" identifier (set by the NT loader, see https://git.reactos.org/?p=reactos.git;a=blob;f=boot/freeldr/freeldr/arch/i386/pc98/pc98hw.c;hb=8e0c920fd2649b1127abe37b88f5753647df5d7d#l1200 ).

Reply:
Moved into the CmpInitializeHardwareConfiguration function. An identifier check would be pointless since all of this information is already known to the kernel